src/charm.py

```diff
@@ -493,4 +493,4 @@ def _get_peer_addresses(self) -> List[str]:
 
 
 if __name__ == "__main__":
-    main(AlertmanagerCharm)
+    main(AlertmanagerCharm, use_juju_for_storage=True)
```

Comment:
Interesting, I never used `use_juju_for_storage`...

Side note here.
@jnsgruk we do not have this parameter documented in [either the SDK or Juju documentation](https://juju.is/docs/search?q=use_juju_for_storage). We only have a docstring in the [Operator framework code](https://operator-framework.readthedocs.io/en/latest/_modules/ops/main.html#main).

Reply:
It's useful... basically always. `_should_use_controller_storage` should set the right default, but for K8s charms, it doesn't hurt anything to be explicit.